tasks/libs/ninja_syntax.py

```diff
@@ -134,6 +134,14 @@ def build(
 
         return outputs
 
+    def phony(
```

Comment:
I think you could do this with `nw.build(rule="phony",...)`

Reply:
Oh you are right, will fix this

Reply:
Should be fixed in last commit